tests/integration/test_s3.py

```diff
@@ -739,7 +744,8 @@ def test_verify_can_switch_sigv4(self):
         service = self.session.get_service('s3')
         endpoint = service.get_endpoint('us-east-1')
         # The set_config_variable should ensure that we use sigv4 for s3.
-        self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
+        # TODO: How do we check this? It's not exposed anywhere anymore...
+        #self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
```

Comment:
What is even the purpose of this test? It is called ``test_can_retrieve_presigned_object`` but it never actually tries to retrieve the object using the presigned url. It just checks that sigv4 was used in the signing.

If its only purpose is to check if sigv4 is used, +1 on removing it. If we actually want to grab the object using the presigned url, we should keep the test and do exactly that.